packages/codegen.go/src/clientFactory.ts

```diff
@@ -87,3 +78,17 @@ export async function generateClientFactory(codeModel: go.CodeModel): Promise<st
   result = contentPreamble(codeModel) + imports.text() + result;
   return result;
 }
+
+export function getAllClientParameters(codeModel: go.CodeModel): Array<go.Parameter> {
```

Comment:
This should probably move to `helpers.js` if it needs to be used in multiple places.

Reply:
moved.